### ISSUE TITLE
VULKAN_LOADER_INSTALL_DIR not used warning

### ISSUE THREAD

**Issue by krOoze** [KhronosGroup/Vulkan-ValidationLayers#174]
So I build all the dependencies like a good puppy.

But `cmake` on this repo claims the `VULKAN_LOADER_INSTALL_DIR` is not used:

> CMake Warning:
>   Manually-specified variables were not used by the project:
> 
>    VULKAN_LOADER_INSTALL_DIR

**Comment by mark-lunarg** [comment]
Hi @krOoze!  This is because building the tests in this repo requires cloning 'googletests' into the external directory (see BUILD.md, excerpted below).  If this isn't present, the test components will intentionally not build and you'd see this message, since only the tests require the loader libraries:

```
cd <the root directory of your copy of the Vulkan-ValidationLayers repository, "external" will be a subdirectory>
git clone https://github.com/google/googletest.git external/googletest
```

@karl-lunarg, I think we talked about defaulting the building of tests to off -- should we do that?

**Comment by karl-lunarg** [comment]
I think that the `set <var> CACHE ...` CMake statement should be moved to the top level CMakeLists.txt file so that the var always gets used whether the user intends to build the tests or not.  That way, it doesn't complain if the user inadvertently supplies the var or if the user always wants to supply the var and control test building by cloning googletest (or not).

**Comment by krOoze** [comment]
@mark-lunarg Thanks! That seems to work.
